lib/recurly.js

```diff
@@ -35,6 +36,10 @@ const defaults = {
   publicKey: '',
   parent: true,
   cors: false,
+  fraud: {
+    dataCollector: false,
```

Comment:
Should be okay if they're in the default here -- it'll deep merge in and this nesting level should be trusted to always be present.